docs/network-requirements.md

```diff
@@ -2,76 +2,44 @@
 
 If your cluster has Internect access, it's all done. Otherwise, If the cluster is behind the outbound proxy or firewall with strict outbound network, make sure following protocols/ports/outbound URLs to function.
 
-- HTTPs on port 443: https://:443
-    - Container registries to host Arc or ML related docker images
-    ```
-    *.azurecr.io
-    *.quay.io
-    auth.docker.io
-    gcr.io
-    quay.io
-    registry-1.docker.io
-    storage.googleapis.com
-    production.cloudflare.docker.com
-    mcr.microsoft.com
-    ```
-    - Azure related services
-    ```
-    *.blob.core.windows.net
-    *.cdn.mscr.io
-    *.data.mcr.microsoft.com
-    *.hcp.*.azmk8s.io
-    *.kusto.windows.net
-    *.queue.core.windows.net
-    *.servicebus.windows.net
-    *.table.core.windows.net
-    *.vault.azure.net
-    *.workspace.<region>.api.azureml.ms
-    <region-code>.his.arc.azure.com
-    <region>.api.azureml.ms
-    <region>.dp.kubernetesconfiguration.azure.com
-    <region>.experiments.azureml.net
-    acs-mirror.azureedge.net
-    apt.dockerproject.org
-    archive.ubuntu.com
-    changelogs.ubuntu.com
-    cloudflare.docker.com
-    data.policy.core.windows.net
-    dc.services.visualstudio.com
-    gbl.his.arc.azure.com
-    go.microsoft.com
-    guestnotificationservice.azure.com
-    login.microsoftonline.com
-    management.azure.com
-    management.azure.com
-    nvidia.github.io
-    onegetcdn.azureedge.net
-    packages.microsoft.com
-    ppa.launchpad.net
-    pypi.org
-    security.ubuntu.com
-    store.policy.core.windows.net
-    sts.windows.net
-    us.download.nvidia.com
-    ```
+### Azure arc
+
+| Destination Endpoint| Port | Use |
+|--|--|--|
+| management.azure.com | https:443 | Required for the agent to connect to Azure and register the cluster. |
+| <region>.dp.kubernetesconfiguration.azure.com | https:443 | Data plane endpoint for the agent to push status and fetch configuration information. |
+| login.microsoftonline.com | https:443 | Required to fetch and update Azure Resource Manager tokens. |
+|  mcr.microsoft.com| https:443 | Required to pull container images for Azure Arc agents. |
+|  *.data.mcr.microsoft.com| https:443 | Required for MCR storage backed by the Azure content delivery network (CDN). |
+|  gbl.his.arc.azure.com| https:443 | Required to get the regional endpoint for pulling system-assigned Managed Service Identity (MSI) certificates. |
+|  <region-code>.his.arc.azure.com| https:443 | Required to pull system-assigned Managed Service Identity (MSI) certificates. |
+| guestnotificationservice.azure.com, sts.windows.net, *.servicebus.windows.net| https:443 | For Cluster Connect and for Custom Location based scenarios. |
+
+### AML extension
+
+| Destination Endpoint| Port | Use |
```

Comment:
for each container registry, could you help explain why they are required, which component/service is relying on it ?